packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -693,44 +693,43 @@ function wait(time) {
 }
 
 function handleErrorMsg(err) {
-  cliux.print(`Error: ${(err?.errorMessage || err?.message) ? err?.errorMessage || err?.message : messageHandler.parse('CLI_EXPORT_CSV_API_FAILED')}`, { color: 'red' })
+  cliux.print(
+    `Error: ${
+      err?.errorMessage || err?.message
+        ? err?.errorMessage || err?.message
+        : messageHandler.parse('CLI_EXPORT_CSV_API_FAILED')
```

Comment:
[MINOR] it could be improved as below mentioned

`(err?.errorMessage || err?.message) ?? messageHandler.parse('CLI_EXPORT_CSV_API_FAILED')`

Reply:
Done